include/utl/logging.h

```diff
@@ -28,6 +28,9 @@
 
 namespace utl {
 
+/**
+  Entrypoint for all Motis logs
+*/
```

Comment:
```suggestion
/// Entrypoint for all logs
```

- I would propose to use comment style (3) from https://www.doxygen.nl/manual/docblocks.html (reason: you can still use `/* ... */` to comment out large blocks of code spanning over multiple doxygen comments)
- I am a fetishist of "fit as much logic into vertical screen space as possible". So I would prefer to waste less lines for comments. One line for this doc string is enough. See the section on "brief description" in the Docxgen docs.